src/OpenTelemetry.Exporter.GoogleCloud/GoogleCloudMetricsExporter.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using Google.Api.Gax.Grpc;
+using Google.Cloud.Monitoring.V3;
+using Google.Cloud.Trace.V2;
+using OpenTelemetry.Exporter.GoogleCloud.Implementation;
+using OpenTelemetry.Metrics;
+
+namespace OpenTelemetry.Exporter.GoogleCloud;
+
+internal sealed class GoogleCloudMetricsExporter : BaseExporter<Metric>
```

Comment:
From a branding perspective, we tend to call this "Google Cloud Monitoring Exporter", e.g see: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/tree/main/exporter/metric